### DISCUSSION TITLE
v11 tag does not include the latest version of v11 range

### DISCUSSION THREAD

**Discussion by ego93** [devsectop/tf-via-pr#301] [General]
Like most actions in GitHub, which have a major-level tag in this case (v11), should include all subversions under the v11 tag, including the latest version within the v11 range.

Unfortunately for this action it is not the case, can this please be fixed?

**Comment by rdhar** [comment]
Hey @ego93, that's a great shout. You should now be able to access v11 tag with all the latest minor and patch releases.

The [new workflow](https://github.com/DevSecTop/TF-via-PR/blob/main/.github/workflows/ci_release.yaml) should automate this process for us going forward for any subsequent major release tags as well.

With that being said, I should call out the security best practice of [pinning to a specific SHA](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions) to harden your CI/CD pipeline against supply chain attacks.
